### ISSUE TITLE
EJS & Eta support

### ISSUE THREAD

**Issue by KaKi87** [alecthomas/chroma#806]
**What problem does this feature solve?**

This feature would solve the lack of syntax highlithing for [`ejs`](https://github.com/mde/ejs) & [`eta`](https://github.com/eta-dev/eta) files.

**What feature do you propose?**

Support these.

---

Thanks

**Comment by KaKi87** [comment]
> @alecthomas closed this as not planned

Why ?

**Comment by alecthomas** [comment]
Because lexers are community driven and written in XML, and It's been 3 years since this was created and there hven't been any takers.

**Comment by KaKi87** [comment]
So you won't take it ?

**Comment by alecthomas** [comment]
No, lexers are maintained by the Chroma community. I don't have the bandwidth.
